src/test/java/io/confluent/examples/streams/TableToTableJoinIntegrationTest.java

```diff
@@ -117,28 +120,15 @@ public void shouldJoinTwoTables() throws Exception {
     streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getAbsolutePath());
 
     StreamsBuilder builder = new StreamsBuilder();
-    KTable<String, String> userRegions = builder.table(stringSerde, stringSerde, userRegionTopic, "user-region-store");
-    KTable<String, Long> userLastLogins = builder.table(stringSerde, longSerde, userLastLoginTopic, "user-last-login-store");
+    KTable<String, String> userRegions = builder.table(userRegionTopic);
+    KTable<String, Long> userLastLogins = builder.table(userLastLoginTopic, Consumed.with(stringSerde, longSerde));
 
-    KTable<String, String> regionAndLastLogin = userRegions.join(userLastLogins,
-        (regionValue, lastLoginValue) -> regionValue + "/" + lastLoginValue);
+    userRegions.join(userLastLogins,
+                     (regionValue, lastLoginValue) -> regionValue + "/" + lastLoginValue,
+                     Materialized.as("joined-store"))
```

Comment:
We should introduce are variable for `"joined-store"` to indicate it's reuse later to query this store.